test-scripts/standalone.sh

```diff
@@ -57,23 +115,16 @@ else
 fi
 
 echo "**************************"
-echo "Running standalone test with "$1 $2 $3 $4
+echo "Running standalone test in $PPM_CONTAINER_NAME container with "$1 $2 $3 $4
 echo "**************************"
 
-#docker stop ppm_kafka > /dev/null
-#docker rm ppm_kafka > /dev/null
-
-# Start the PPM in a new container.
-docker run --name ppm_kafka -v /tmp/docker-test/data:/ppm_data -it --rm -p '8080:8080' -d jpocvdp_ppm:latest /cvdi-stream/docker-test/ppm_standalone.sh > /dev/null
-
-sleep 10
+startPPMContainer
 
+# Produce the test data.
 if [ $4 = "BSM" ]; then
-    docker exec ppm_kafka /cvdi-stream/docker-test/do_bsm_test.sh $OFFSET
-    # Produce the test data.
+    docker exec $PPM_CONTAINER_NAME /cvdi-stream/docker-test/do_bsm_test.sh $OFFSET
```

Comment:
The same here. It might be helpful to use relative path rather than absolute path.

Reply:
Since the command is intended to run inside of a docker container, an absolute path should be a fine method for referencing the script. I don't think this is a problem.